src/codeu/chat/client/ClientUser.java

```diff
@@ -65,14 +74,15 @@ public User getCurrent() {
     return current;
   }
 
-  public boolean signInUser(String name) {
+  public boolean signInUser(String name, String password) {
```

Comment:
I can see what you mean, but most of the code was previously from the initial CodeU draft. With little time remaining and other things as priority, I think I'm going to put this on the back-burner for now.